File: Midterm/Loop/06. Prime Number/prime.c
//check whether a number is Prime number or not
#include<stdio.h>
int main()
{
    int n,i,flag=0;
    printf("Enter the number\n");
    scanf("%d",&n);
    for(i=2;i<=n/2;i++)
    {
        if(n%i==0)
        {
            flag=1;
            break;
        }
    }
    if(flag==1)
    {
        printf("%d is not a prime number\n",n);
    }
    else
    {
        printf("%d is a prime number\n",n);
    }
    return 0;
}

File: 4th Semester/Prime.java
// Prime number or not?

public class Prime {

    public static void main(String[] args) {

        int num = 25;
        int flag = 0;
        for (int i = 2; i <= num / 2; ++i) {
            // condition for nonprime number
            if (num % i == 0) {
                flag = 1;
                break;
            }
        }

        if (flag==0)
            System.out.println(num + " is a prime number.");
        else
            System.out.println(num + " is not a prime number.");
    }
}